static/docs/user-guide/dvcignore.md

```diff
@@ -13,17 +13,17 @@ files handling. `.dvcignore` by design works similar way as `.gitignore` does.
 
 - You need to create `.dvcignore` file.
 - Populate it with [patterns](https://git-scm.com/docs/gitignore) that you would
-  like to ignore
+  like to ignore.
 - Each line should contain only one pattern.
 - During execution of commands that traverse directories, DVC will ignore
   matching paths.
-- Ignored files will not be cached, they will be non-existent for DVC. It's
-  worth to remember that, especially when ignoring files inside DVC-handled
-  directories.
-- Keep in mind, that when you add to dvcignore entries that affect one of the
+- Ignored files will not be saved in cache, they will be non-existent for DVC.
```

Comment:
we should make it clear with some note that DVC removes ignored files on `dvc repro`, `dvc run`, etc. It's super important to understand I think. And it's not clear from the description now.